From 44bf7868e5c2c4a52aef67e55aab1e904147dad4 Mon Sep 17 00:00:00 2001
From: Kent Overstreet <kent.overstreet@linux.dev>
Date: Fri, 29 Dec 2023 20:16:37 -0500
Subject: [PATCH] fix missing atomic64_read_acquire on 32 bit

Signed-off-by: Kent Overstreet <kent.overstreet@linux.dev>
---
 include/linux/atomic.h | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/include/linux/atomic.h b/include/linux/atomic.h
index 7effc16..7f76087 100644
--- a/include/linux/atomic.h
+++ b/include/linux/atomic.h
@@ -300,6 +300,13 @@ DEF_ATOMIC_OPS(atomic_long,	long)
 DEF_ATOMIC_OPS(atomic64,	s64)
 #else
 s64 atomic64_read(const atomic64_t *v);
+s64 atomic64_read_acquire(const atomic64_t *v)
+{
+	s64 ret = atomic64_read(v);
+	smp_mb__after_atomic();
+	return ret;
+}
+
 void atomic64_set(atomic64_t *v, s64);
 
 s64 atomic64_add_return(s64, atomic64_t *);
-- 
2.43.0

