From 1ff643143d6724964ffc1a6f762e4ffd64566ed4 Mon Sep 17 00:00:00 2001
From: copyliu <copyliu@gmail.com>
Date: Mon, 10 Aug 2026 09:42:29 +0800
Subject: [PATCH] Makefile: Do not run Rust builds

---
 Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 8b11903e8..fd42bfcce 100644
--- a/Makefile
+++ b/Makefile
@@ -271,7 +271,7 @@ BCACHEFS_DEPS=libbcachefs.a
 RUST_SRCS:=$(shell find src fs bch_bindgen bcachefs-shim -type f ! -path 'fs/vendor/kernel-rust/*' -iname '*.rs') build.rs
 
 bcachefs: $(BCACHEFS_DEPS) $(RUST_SRCS)
-	$(Q)$(CARGO_BUILD)
+	@echo "    [RUST]   Skipped"
 
 libbcachefs.a: $(OBJS)
 	@echo "    [AR]     $@"
@@ -458,7 +458,6 @@ clean:
 	@echo "Cleaning all"
 	$(Q)$(RM) libbcachefs.a c_src/libbcachefs.a .version dkms/dkms.conf build.vars *.tar.xz $(DOCGENERATED)
 	$(Q)$(RM) -r build
-	$(Q)$(CARGO_CLEAN)
 	$(Q)$(RM) -f $(built_scripts)
 
 .PHONY: deb
-- 
2.47.3

