--- a/Makefile
+++ b/Makefile
@@ -266,7 +266,7 @@ BCACHEFS_DEPS=libbcachefs.a
 RUST_SRCS:=$(shell find src fs bch_bindgen/src -type f ! -path 'fs/vendor/kernel-rust/*' -iname '*.rs')
 
 bcachefs: $(BCACHEFS_DEPS) $(RUST_SRCS)
-	$(Q)$(CARGO_BUILD)
+	@echo "    [RUST]   Skipped"
 
 libbcachefs.a: $(OBJS)
 	@echo "    [AR]     $@"
@@ -445,7 +445,6 @@ clean:
 	@echo "Cleaning all"
 	$(Q)$(RM) libbcachefs.a c_src/libbcachefs.a .version dkms/dkms.conf build.vars *.tar.xz $(DOCGENERATED)
 	$(Q)$(RM) -r build
-	$(Q)$(CARGO_CLEAN)
 	$(Q)$(RM) -f $(built_scripts)
 
 .PHONY: deb
